app/src/main/res/layout/activity_feedback.xml

```diff
@@ -13,4 +13,41 @@
         android:layout_height="match_parent"
         app:dlfvCameraOrnament="@drawable/dragonfly_lens_grid"
         app:dlfvCameraOrnamentScaleType="center"/>
+
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        style="@style/AppTheme.Toolbar"
+        android:elevation="@dimen/toolbar_elevation">
+
+        <RelativeLayout
```

Comment:
If I enable the home button, it splits the toolbar in two. Then, in order to centralize the title, I would have to use a negative margin (to ignore the size of the area dedicated to the home button). Do you think it's better?